devito/operator/operator.py

```diff
@@ -366,30 +387,42 @@ def _specialize_iet(cls, graph, **kwargs):
         """
         return graph
 
+    @classmethod
+    @timed_pass(name='lowering.uIET')
+    def _lower_uiet(cls, stree, profiler=None, **kwargs):
```

Comment:
Note for reviewers: new thin layer between STree and IET necessary to fully take advantage of the recursive compilation feature mentioned earlier